ncoreparser/parser.py

```diff
@@ -9,13 +9,18 @@ class TorrentsPageParser:
     def __init__(self):
         self.type_pattern = re.compile(r'<a href=".*\/torrents\.php\?tipus=(.*?)">'
                                        r'<img src=".*" class="categ_link" alt=".*" title=".*">')
-        self.id_name_pattern = re.compile(r'<a href=".*?" onclick="torrent\(([0-9]+)\); return false;" title="(.*?)">')
+        self.id_name_poster_pattern = re.compile(
```

Comment:
I recommend to use simpler regex here. As I see there is the image tag:
```
<img onmouseout="elrejt('borito3814785')" onmouseover="mutat('https://nc-img.cdn.l7cache.com/covers/L9_kjpbQ9UwZFlXZ?28795815', '281', 'borito3814785', this)" border="0" src="data:image/gif;base64,R0lGODlhDwAPAJEAAAAAAP///////wAAACH5BAEAAAIALAAAAAAPAA8AAAINlI+py+0Po5y02otnAQA7" class="infobar_ico">
```
And here you could parse out the torrent id: `elrejt('borito3814785')` the id is: 3814785.

Reply:
Then you can iterate over the ids. check is the id in this pattern output. If yes add if not set no cover.